src/components/SolFeesApp.jsx

```diff
@@ -6,29 +6,47 @@ import Image from 'next/image';
 import { useEffect, useLayoutEffect, useState } from 'react'
 import Confetti from 'react-confetti';
 import { useMeasure } from 'react-use';
+import { walletNameToAddressAndProfilePicture } from '@portal-payments/solana-wallet-names';
```

Comment:
Please rename the import to something more feasible, maybe `fetchDomainDetails`?